services/core-api/app/api/projects/project_decision_package/models/project_decision_package.py

```diff
@@ -142,8 +142,6 @@ def update(self,
             self.save(commit=False)
 
         if len(documents) > 0:
-            mine_document_guid = documents[0].mine_document_guid
```

Comment:
this is/was a bug in project decision package.
It's a new document, so it doesn't have a mine_document_guid. Also, project is passed in as a parameter so it doesn't need to be queried below. 

There was a spot to show project decision docs on the MMA and I wanted to see it populated, and this was a surprisingly quick fix. In the end, with discussion with Rebecca, it's hidden behind a feature flag (on the FE it was a document table with documents=[], so not at all useful)